### PR TITLE
#766 Modified confidence filter to display results by high confidence by default

### PR DESCRIPTION
**Pull request by bhekidaweti**
Updated filters to sort by confidence level to high by default

### CODE REVIEW

**Comment by hibaa03** [comment]
HI, for the CI tests failing read [this](https://github.com/mozilla/perfcompare?tab=readme-ov-file#running-all-validators-and-tests) part of the README.

**Comment by bhekidaweti** [comment]
Thanks, I am working on it.

File: src/components/CompareResults/ResultsTable.tsx [hunk 1]
```diff
@@ -104,9 +104,17 @@ export default function ResultsTable() {
...
+    new Map([['confidence', new Set(['High'])]]), // ColumnID -> Set<Values to remove>// Default confidence filter
   );
 
+  useEffect(() => {
+    setTableFilters((prevFilters) => {
+      const newFilters = new Map(prevFilters);
+      newFilters.set('confidence', new Set(['High']));
+      return newFilters;
+    });
+  }, []);
```
Comment (julienw) [inline src/components/CompareResults/ResultsTable.tsx:116]: can you please explain why you used this effect in addition to the default state above?
Thanks

File: src/components/CompareResults/ResultsTable.tsx [hunk 2]
```diff
@@ -104,9 +104,17 @@ export default function ResultsTable() {
   const [searchTerm, setSearchTerm] = useState(initialSearchTerm);
   const [frameworkIdVal, setFrameworkIdVal] = useState(frameworkId);
   const [tableFilters, setTableFilters] = useState(
-    new Map() as Map<string, Set<string>>, // ColumnID -> Set<Values to remove>
+    new Map([['confidence', new Set(['High'])]]), // ColumnID -> Set<Values to remove>// Default confidence filter
```
Comment (julienw) [inline src/components/CompareResults/ResultsTable.tsx:107]: Have you tried the code locally? This does the opposite of what you think: this _removes_ all items with the "high" confidence, we want to do the opposite.

**Review by julienw — CHANGES_REQUESTED** [review]
thanks for your contribution,
please note that the code does the opposite of what's needed: it removes all results with the "high" confidence instead of keeping them only.

Thanks!

**Pull request closed without merging**